coalib/output/Logging.py

```diff
@@ -41,37 +41,64 @@ def configure_logging():
     # reset counter handler
     CounterHandler.reset()
 
-    logging.config.dictConfig({
-        'version': 1,
-        'handlers': {
-            'colored': {
-                'class': 'logging.StreamHandler',
-                'formatter': 'colored',
-                'stream': sys.stderr
+    if not no_color:
```

Comment:
this looks too complicated to build up two completely different dicts just because of this param. Can't you modify only the entries of the current dict that would be affected by `no-color = True`? As far as I see we have to add the `plain` handler and formatter to the config and just use it for `root.handlers` instead of the `colored`-handler.

If you really don't want to create the color formatters if `no-color` is given, you should try to build upon a dict with common settings. In this case only the version is common, but at least we don't duplicate information:
```
config = {'version': 1}

if not_color:
    # I don't know a better name now for this variable...
    print_config = ...
else:
    print_config = ...

config.update(print_config)
logging.config.dictConfig(config)
```